pkg/utils/leafselection.go

```diff
@@ -37,8 +39,13 @@ func LeafSelection(ctx context.Context, configAdminServiceClient admin.ConfigAdm
 		if strings.HasPrefix(qpp, "{") {
 			pathID := args[nextArgIdx]
 			prevPart := newQueryParts[len(newQueryParts)-1]
-			newQueryParts[len(newQueryParts)-1] =
-				fmt.Sprintf("%s[%s=%s]", prevPart, pathID.Name, pathID.Value)
+			if pathID.Value != "new" {
```

Comment:
@SeanCondon We probably don't want this anymore.

Reply:
yeah - can remove